### ISSUE TITLE
Add Dockerfile for AssetAdmistrationShellRepository 

### ISSUE THREAD

**Issue by bjoernhoeper** [eclipse-basyx/basyx-dotnet-applications#2]
The Registry Server is already dockerized but a Dockerfile for the AssetAdministrationShellRepository is not yet implemented.

**Comment by HansG89** [comment]
All of the basyx dotnet applications can be dockerized in the same way, by simply adding a Dockerfile like this:

```
FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
WORKDIR /src
COPY AasServer.csproj .
RUN dotnet restore AasServer.csproj
COPY .
RUN dotnet publish AasServer.csproj -c Release -o /app/publish

FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS runtime
WORKDIR /app
COPY --from=build /app/publish .
EXPOSE 5998
ENTRYPOINT ["dotnet", "AasServer.dll"]
```

**Comment by bjoernhoeper** [comment]
Yes I know but I think there should be one file for each of these in the repository. So the issue is more a work item to do that. And when that is done I would propose to also automate building the images and pushing them to a central registry. Mainly to foster adaptation by other users.

> All of the basyx dotnet applications can be dockerized in the same way, by simply adding a Dockerfile like this:
> 
> ```
> FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
> WORKDIR /src
> COPY AasServer.csproj .
> RUN dotnet restore AasServer.csproj
> COPY .
> RUN dotnet publish AasServer.csproj -c Release -o /app/publish
> 
> FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS runtime
> WORKDIR /app
> COPY --from=build /app/publish .
> EXPOSE 5998
> ENTRYPOINT ["dotnet", "AasServer.dll"]
> ```
